spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -81,570 +81,567 @@
 @RequestMapping(value = "/admin")
 public class AdminController {
 
-    private static final Logger log = LoggerFactory.getLogger(AdminController.class);
+	private static final Logger log = LoggerFactory.getLogger(AdminController.class);
```

Comment:
It looks like spaces were replaced by tabs in this file, would be good to switch back to "4 spaces" instead of tabs, like in the other Coatrack source files

Reply:
fixed